vendor/vendor.json

```diff
@@ -3,10 +3,52 @@
 	"ignore": "test",
 	"package": [
 		{
-			"checksumSHA1": "TVyK4buHhPUkpD0Pt9M7Zo2UAlI=",
+			"path": "appengine",
+			"revision": ""
+		},
+		{
+			"path": "appengine/aetest",
+			"revision": ""
+		},
+		{
+			"path": "appengine/urlfetch",
+			"revision": ""
+		},
+		{
+			"checksumSHA1": "hA3MoSjATaM7AQknoqXcE5skk00=",
 			"path": "github.com/Financial-Times/go-fthealth/v1_1",
-			"revision": "bc27ed19189994eef0364e4a07879d5a3f3be76f",
-			"revisionTime": "2017-03-24T12:21:32Z"
+			"revision": "e7ccca038327a0091303a52445ec1f0fb66cb97b",
+			"revisionTime": "2017-05-25T09:50:41Z"
+		},
+		{
+			"checksumSHA1": "4gy/7n/lgtdi5efuHXwFcIe1ZI8=",
```

Comment:
Any of our libraries that have versions (there are a couple that don't) should be vendored with the version information, not just the revision.